test/e2e/helm/e2e_suite_test.go

```diff
@@ -14,5 +14,6 @@ func TestE2E(t *testing.T) {
 }
 
 var _ = Describe("Test Zone and Global with Helm chart", Label("job-2"), Label("arm-not-supported"), helm.ZoneAndGlobalWithHelmChart, Ordered)
+var _ = Describe("Test Zone and Global universal mode with Helm chart", Label("job-2"), Label("arm-not-supported"), helm.ZoneAndGlobalInUniversalModeWithHelmChart, Ordered)
```

Comment:
pick job-0. See distribution in master
https://app.circleci.com/pipelines/github/kumahq/kuma/19346/workflows/7b06befa-1ce7-4462-8fb1-1abf9b32ddea/jobs/318436